src/components/navigation/components/DrawerNavigation/index.tsx

```
@@ -4,7 +4,7 @@ import { AnimatePresence, motion } from 'framer-motion'
 import { X } from 'lucide-react'
 import { useCallback, useEffect, useState } from 'react'
 import type React from 'react'
-import { useGesture } from 'react-use-gesture'
+import { useDrag } from '@use-gesture/react'
```

ESLint

[error] `@use-gesture/react` import should occur before import of `framer-motion`

(import/order)